src/modules/configs/templates/application.py

```diff
@@ -0,0 +1,19 @@
+config = {
+    "containers": {
+        "sata": {
+            "device_path": None,
+            "mount_path": None
+        }
+    },
+    "resources": {
+        "thread_count": None
+    },
+    "workdir": None,
```

Comment:
For example: This field shouldn't be mandatory - it can be defined in all 3 confs.